homeassistant/components/hassio/sensor.py

```diff
@@ -27,11 +27,13 @@
         entity_registry_enabled_default=False,
         key=ATTR_VERSION,
         name="Version",
+        icon="mdi:package-up",
```

Comment:
`mdi:package-up` is an icon we generally use on binary sensors to indicate an actual update. I think that that is not matching for a sensor providing the currently installed version.

Reply:
I also don't think it's the best option, I just wanted to keep the one already used in [`sensor.py`](https://github.com/home-assistant/core/blob/f344ea7bbba974c7592a1dcde8cfa7ccc71f3f0d/homeassistant/components/version/sensor.py#L97) from the version integration.

What do you think of these options?

Version:
* `mdi:checkbox-marked-circle-outline` ![image](https://user-images.githubusercontent.com/31328123/153718136-ca0e731f-1c64-4df1-ae8f-f8d17064c6b9.png)

* `mdi:checkbox-marked-outline` ![image](https://user-images.githubusercontent.com/31328123/153718209-d11dade3-6635-4bdc-97a1-5c43a20a89e7.png)

Newest Version:
* `mdi:new-box` ![image](https://user-images.githubusercontent.com/31328123/153717986-731a46ce-37d8-4d64-bd54-912e8fc839be.png)
* `mdi:update` ![image](https://user-images.githubusercontent.com/31328123/153718075-cc3ac14a-de2a-4c59-9cdf-61808ce60e60.png)

Do you have any suggestions? I just didn't want to keep the default icon... ![image](https://user-images.githubusercontent.com/31328123/153718326-e18b61db-d23d-4198-9431-c214ac6cdee6.png)

Reply:
- `apcsupsd`  uses `mdi:information-outline`
- `dsmr_reader` uses `mdi:alert-circle`
-  `edl21` uses `mdi:flash`
- `minecraft_server` uses `mdi:numeric`
- `system_bridge` uses `mdi:chip` and `mdi:counter`
- `version` uses `mdi:package-up`
- 
There are most likely some I missed as well when scanning.
Maybe it's time to add a "version" device class with a standard icon.

Reply:
So can we go on to create the device class `version`? Can the icon be `mdi:information-outline`?

The core and documentation part I believe I can do it, as for the frontend part I may need some help.

Reply:
The first step would be to open an architectural discussion under the "Entity Models" category about it <https://github.com/home-assistant/architecture/discussions>